lib/lotus/helpers/html_helper/empty_html_node.rb

```diff
@@ -48,7 +54,12 @@ def attributes
           result = [nil]
 
           @attributes.each do |name, value|
-            result << %(#{ name }="#{ value }")
+            # only present shortform `required` on certain elements
+            if name.to_s == 'required' && HTML5_REQUIRED_CONSTRAINT.include?(@name.to_s)
```

Comment:
If _value_ is _false_ then the whole attribute needs to be omitted. This is currently not happening.
I'd split this in two methods (for normal attributes, for boolean attributes) and add a test for the new case..